src/openforms/config/models.py

```diff
@@ -283,6 +284,59 @@ class GlobalConfiguration(SingletonModel):
         help_text=_("When enabled, information about the used SDK is displayed."),
     )
 
+    # Removing data configurations
+    successful_submissions_removal_limit = models.PositiveIntegerField(
+        _("successful submission removal limit"),
+        default=7,
+        validators=[MinValueValidator(1)],
+        help_text=_(
+            "Amount of days successful submissions will remain before being removed"
+        ),
+    )
+    successful_submissions_removal_method = models.CharField(
+        _("successful submissions removal method"),
+        max_length=50,
+        choices=RemovalMethods,
+        default=RemovalMethods.delete_permanently,
+        help_text=_("How successful submissions will be removed after the limit"),
+    )
+    incomplete_submissions_removal_limit = models.PositiveIntegerField(
+        _("incomplete submission removal limit"),
+        default=7,
+        validators=[MinValueValidator(1)],
+        help_text=_(
+            "Amount of days incomplete submissions will remain before being removed"
+        ),
+    )
+    incomplete_submissions_removal_method = models.CharField(
+        _("registration backend status"),
+        max_length=50,
+        choices=RemovalMethods,
+        default=RemovalMethods.delete_permanently,
+        help_text=_("How incomplete submissions will be removed after the limit"),
+    )
+    errored_submissions_removal_limit = models.PositiveIntegerField(
+        _("errored submission removal days limit"),
```

Comment:
```suggestion
        _("errored submission removal limit"),
```
days is not added to the other labels, so lets not add it here suddenly.